ToolkitApp/tabs/src/components/sample/Welcome.css

```diff
@@ -7,25 +7,25 @@
   padding: 4rem;
 }
 
-.welcome.page > .narrow > img {
+.welcome.page>.narrow>img {
   margin: 0 auto;
   display: block;
   width: 200px;
 }
 
-.welcome.page > .narrow > ul {
+.welcome.page>.narrow>ul {
   width: 80%;
   justify-content: space-between;
   margin: 4rem auto;
   border-bottom: none;
 }
 
-.welcome.page > .narrow > ul > li {
+.welcome.page>.narrow>ul>li {
   background-color: inherit;
   margin: auto;
 }
 
-.welcome.page > .narrow > ul > li > a {
```

Comment:
instead of using a long list of selector, you could specify the style in a class name like
```
.my-class {
      font-size: 14px;
      min-height: 32px;
      border-bottom-color: rgb(98, 100, 167);
      display: 'flex';
}
```
 and apply it directly to the html element `<a class="my-class">abc</a>`

Reply:
Try to give your css descriptive and meaningful class names so others could reuse them :)

Reply:
FYI: I'll be cleaning up the CSS in a later PR, as the CSS is very messy